mainnet-1/specs/cosmoshub.json

```diff
@@ -0,0 +1,159 @@
+{
+  "proposal": {
+    "title": "Add Specs: Cosmos hub",
+    "description": "Adding new specification support for relaying cosmoshub data on Lava",
+    "specs": [
+      {
+        "index": "COSMOSHUB",
+        "name": "cosmos hub mainnet",
+        "enabled": true,
+        "imports": [
+          "COSMOSSDKFULL",
+          "COSMOSSDK45DEP"
+        ],
+        "reliability_threshold": 268435455,
+        "data_reliability_enabled": true,
+        "block_distance_for_finalized_data": 0,
+        "blocks_in_finalization_proof": 1,
+        "average_block_time": 6500,
+        "allowed_block_lag_for_qos_sync": 2,
+        "shares": 1,
+        "min_stake_provider": {
+          "denom": "ulava",
+          "amount": "47500000000"
```

Comment:
47.5k LAVA is too high, let's do 15K LAVA as a standard for iprpc chains

Reply:
👍